tests/integration.rs

```diff
@@ -120,6 +121,20 @@ mod cache_tests {
         exec(CACHE_RUST_TEST, data, add_imports)
     }
 
+    #[test]
+    fn test_tikv_get_set() -> Result<()> {
+        init();
+
+        let data = Some(TikvClient::new("127.0.0.1:2379")?);
```

Comment:
I assume this requires running a database instance locally?
Have a look at how the Redis test controller works to automatically start it.

Reply:
Yes I will add a controller to start the tikv cluster